src/eos_kolibri/eos_kolibri_migrate/main.py

```diff
@@ -16,119 +16,250 @@
+            )
+            needs_backup = True
+        else:
+            print(
+                "Your personal Kolibri home will be moved to the system-wide Kolibri home at '{target_path}'".format(
+                    target_path=self.target_path.as_posix()
+                )
+            )
+            needs_backup = False
+
+        print()
+
+        answer = self.ask_yes_no(
+            "Would you like to replace the system-wide Kolibri home with your own Kolibri home?",
+            default=not needs_backup,
+        )
+
+        if answer:
+            return self.__do_copy_source_to_target(needs_backup=needs_backup)
+        else:
+            return 0
 
-    recursive_chown(target_path, kolibri_pwd.pw_uid, kolibri_pwd.pw_gid)
+    def __do_copy_source_to_target(self, needs_backup):
+        print("Stopping Kolibri...")
+        # stop_kolibri_services()
+        terminate_kolibri_server_process()
 
-    # Restart the Kolibri socket service.
-    start_kolibri_services()
+        target_stat = self.target_path.stat()
+        working_path = get_backup_path(self.target_path, ".new")
+        if needs_backup:
+            backup_path = get_backup_path(self.target_path)
+        else:
+            backup_path = None
+
+        print("Copying files...")
+        shutil.copytree(self.source_path, working_path)
+        recursive_chown(working_path, target_stat.st_uid, target_stat.st_gid)
+        if backup_path:
+            self.target_path.rename(backup_path)
+        working_path.replace(self.target_path)
+
+        print("Restarting Kolibri...")
+        # start_kolibri_services()
+
+        print()
 
-    logging.info("Successfully moved Kolibri data to {}".format(target_path))
+        print(
+            "Successfully copied personal Kolibri home to '{target_path}'".format(
+                target_path=self.target_path.as_posix()
+            )
+        )
+
+        print()
+
+        if backup_path:
+            print(
+                "The original system-wide Kolibri home has been backed up to '{backup_link}'".format(
+                    backup_link=self.__path_to_ansii_link(backup_path)
+                )
+            )
+
+        return self.__do_move_source_to_backup()
+
+    def __do_move_source_to_backup(self):
+        backup_path = get_backup_path(self.source_path)
+        self.source_path.rename(backup_path)
+
+        print(
+            "Your original personal Kolibri home has been backed up to '{backup_link}'".format(
+                backup_link=self.__path_to_ansii_link(backup_path)
+            )
+        )
+
+        print()
+
+        print("You are now using system-wide Kolibri")
+
+        return 0
+
+    def __path_to_ansii_link(self, path):
+        if self.interactive:
+            path_str = path.as_posix()
+            uri_str = path.as_uri()
+            return f"{ANSII_ESC_LINK}{uri_str}{ANSII_BEL}{path_str}{ANSII_ESC_LINK}{ANSII_BEL}"
+        else:
+            return path.as_posix()
```

Comment:
If using [click.echo](https://click.palletsprojects.com/en/7.x/api/#click.echo) it will take care of hiding ANSI codes automatically if the destination file is not a terminal.